### ISSUE TITLE
Reports Preview/Mails show no graphs

### ISSUE THREAD

**Issue by sam-san91** [Cacti/cacti#1852]
Updated to the latest develop version and the reports don't work anymore.
The graphs won't get displayed in neither the preview nor the mail that gets sent out.

That's shown in the logs:
```
[Mon Aug 20 16:35:39.695671 2018] [php7:notice] [pid 2671] PHP Notice:  Undefined index: policy_graphs in /usr/share/cacti/lib/auth.php on line 1062, referer: http://cacti.my.local/reports_admin.php?action=edit&tab=details&id=1&page=1
[Mon Aug 20 16:35:39.695882 2018] [php7:notice] [pid 2671] PHP Notice:  Undefined index: type in /usr/share/cacti/lib/auth.php on line 1068, referer: http://cacti.my.local/reports_admin.php?action=edit&tab=details&id=1&page=1
[Mon Aug 20 16:35:39.695991 2018] [php7:notice] [pid 2671] PHP Notice:  Undefined index: type in /usr/share/cacti/lib/auth.php on line 1068, referer: http://cacti.my.local/reports_admin.php?action=edit&tab=details&id=1&page=1
[Mon Aug 20 16:35:39.696111 2018] [php7:notice] [pid 2671] PHP Notice:  Undefined index: id in /usr/share/cacti/lib/auth.php on line 1068, referer: http://cacti.my.local/reports_admin.php?action=edit&tab=details&id=1&page=1
[Mon Aug 20 16:35:39.696213 2018] [php7:notice] [pid 2671] PHP Notice:  Undefined index: type in /usr/share/cacti/lib/auth.php on line 1069, referer: http://cacti.my.local/reports_admin.php?action=edit&tab=details&id=1&page=1
[Mon Aug 20 16:35:39.696313 2018] [php7:notice] [pid 2671] PHP Notice:  Undefined index: policy_hosts in /usr/share/cacti/lib/auth.php on line 1072, referer: http://cacti.my.local/reports_admin.php?action=edit&tab=details&id=1&page=1
[Mon Aug 20 16:35:39.696437 2018] [php7:notice] [pid 2671] PHP Notice:  Undefined index: type in /usr/share/cacti/lib/auth.php on line 1078, referer: http://cacti.my.local/reports_admin.php?action=edit&tab=details&id=1&page=1
[Mon Aug 20 16:35:39.696540 2018] [php7:notice] [pid 2671] PHP Notice:  Undefined index: type in /usr/share/cacti/lib/auth.php on line 1078, referer: http://cacti.my.local/reports_admin.php?action=edit&tab=details&id=1&page=1
[Mon Aug 20 16:35:39.696667 2018] [php7:notice] [pid 2671] PHP Notice:  Undefined index: id in /usr/share/cacti/lib/auth.php on line 1078, referer: http://cacti.my.local/reports_admin.php?action=edit&tab=details&id=1&page=1
[Mon Aug 20 16:35:39.696777 2018] [php7:notice] [pid 2671] PHP Notice:  Undefined index: type in /usr/share/cacti/lib/auth.php on line 1079, referer: http://cacti.my.local/reports_admin.php?action=edit&tab=details&id=1&page=1
```
Not quite sure where the problem is, though

**Comment by sam-san91** [comment]
Ahh. This has nothing to do with the latest version. The user who generated the report has been deleted.
Sorry :(

**Comment by netniV** [comment]
I would say that is still an issue.  The report shouldn't fail just because the original user is no longer present.  I'm not sure why that would be a requirement other than to say who created it.  If they aren't present, that should simply be 'Unknown'

**Comment by cigamit** [comment]
I'm going to re-open this one.  We should disable the issue and provlde some sort of status for this issue.

**Comment by cigamit** [comment]
Resolved.  This will be included in the 1.2 release.
